OsmAnd-java/src/main/java/net/osmand/router/MissingMapsCalculationResult.java

```diff
@@ -14,6 +14,7 @@ public class MissingMapsCalculationResult {
 
 	private final RoutingContext missingMapsRoutingContext;
 	private final List<LatLon> missingMapsPoints;
+	private boolean onlineResultPoints = false;
```

Comment:
This had bad name & bad intention, I would like to understand the gist of this field and why it's necessary here

Reply:
refactored